src/libraries/System.Text.Json/tests/Common/TestClasses/TestClasses.Constructor.cs

```diff
@@ -1972,6 +1972,34 @@ public void VerifyMinimal()
             MyTuple.Item4.Verify();
         }
     }
+    public class Class_With_Parameters_Default_Values
+    {
+        public int I { get; }
+        public float F { get;}
+        public double D { get;}
+        public decimal M { get;}
+
+        public Class_With_Parameters_Default_Values(int i = 21, float f = 42.0f, double d = 3.14159, decimal m = 3.1415926535897932384626433M)
```

Comment:
Could you also add a char parameter?

Reply:
Could you also add nullable variants of the same parameter types?

Reply:
Not sure if we're testing for this already elsewhere, but could you also add an optional enum parameter?

Reply:
Added these cases